signac/contrib/project.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, config=None):
 
         # Ensure that the project id is configured.
         try:
-            self._id = str(self.config["project"])
+            self.id_ = str(self.config["project"])
```

Comment:
Similar to the job, this private attribute may need to be changed back from `self.id_` to `self._id`. I think the cases that we want to rename are mostly local variables or function parameters named `_id`.